MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -255,8 +258,11 @@ namespace WPEFramework {
             }
 
             LOGINFO("Reboot_Pending :%s",g_is_reboot_pending.c_str());
-
-            string cmd="";
+            /* Delay the network call to make sure the network plugins is activated
+	     * before making any call to network plugins
+	     */
+            sleep(30);
```

Comment:
Why are we adding 30 seconds delay everytime? This is going to delay every maintenance activity thread.
Pls check if network plugin is activated before adding delay.